.github/workflows/manual-package.yml

```diff
@@ -66,4 +65,34 @@ jobs:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           clientid: ${{ secrets.SFDX_AUTH_CLIENT_ID }}
           jwtkey: ${{ secrets.SFDX_AUTH_JWT_KEY }}
-  
+  bump-version:
+    name: 'Bump version'
+    needs:
+      - 'lint-check'
+      - 'unit-test'
+      - 'e2e-atomic-test'
+      - 'e2e-quantic-test'
+    runs-on: ubuntu-latest
+    if: always()
+    steps:
+      - name: fail if unit test failed
+        if: ${{ needs.unit-test.result == 'failure' }}
+        run: exit 1
+      - name: fail if atomic e2e failed
+        if: ${{ needs.e2e-atomic-test.result == 'failure' }}
+        run: exit 1
+      - name: fail if quantic e2e failed
+        if: ${{ needs.e2e-quantic-test == 'failure' }}
+        run: exit 1
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.ADMIN_GITHUB_TOKEN }}
+          fetch-depth: 0
+      - uses: ./.github/actions/setup
+      - name: Clean directory
+        run: git clean -xfd -e node_modules/ -e .husky
+      - name: Commit & push
+        run: |
+          git config --global user.name "GitHub Actions Bot"
+          git config --global user.email "<>"
+          npm run bump:version
```

Comment:
I think this should be extracted into a composite action to avoid having to remember two places to update the version bump steps.

Careful, composite actions don't support environment variables, so the GitHub token would need to be passed as an input just like `e2e-atomic`.